src/EloquentDataTable.php

```diff
@@ -38,6 +45,9 @@ public function __construct($model)
         parent::__construct($builder->getQuery());
 
         $this->query = $builder;
+
+        $connection   = config('database.default');
```

Comment:
There are cases where the connection is set from the model or via runtime using query. This is will not work on those situations.

Reply:
Ok, sure. But in your earlier reply (https://github.com/yajra/laravel-datatables/pull/2051#discussion_r274738567) you said, basically, this wouldnt even be necessary. So, what now?

Reply:
Yes, so please revert the commits to 1st one you sent. Might be able to test this within the week. Thanks!

Reply:
Ok, done. Sorry for the mess with the commits.